lib/common/output_text.c

```diff
@@ -26,6 +21,12 @@ typedef struct text_private_s {
     GQueue *parent_q;
 } text_private_t;
 
+static pcmk__output_option_t text_options[] = {
+    { "fancy-output", "0", "Choose between simple and highly formatted output" },
```

Comment:
If we're going to go with --output-NAME as the user interface, we probably want to rename this :)

Reply:
Yeah, that had occurred to me too.